robot/Cumulus/tests/browser/relationships/create_relationships.robot

```diff
@@ -1,48 +1,69 @@
 *** Settings ***
 
 Resource        robot/Cumulus/resources/NPSP.robot
-Suite Setup     Open Test Browser
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/AccountPageObject.py
+...             robot/Cumulus/resources/RelationshipPageObject.py
+...             robot/Cumulus/resources/NPSP.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
 Suite Teardown  Delete Records and Close Browser
-Test Teardown   Capture Screenshot on Failure
+
+***Keywords***
+
+Capture Screenshot on Failure
+    Run Keyword If Test Failed  Capture Page Screenshot
```

Comment:
Remove this if you are not using it anymore

Reply:
Removed